pkg/quarterdeck/db/models/users.go

```diff
@@ -345,9 +346,37 @@ func ListUsers(ctx context.Context, orgID any, prevPage *pagination.Cursor) (use
 	}
 	defer tx.Rollback()
 
+	// Build parameterized query with WHERE clause
+	var query strings.Builder
+	query.WriteString(getUserSQL)
+
+	// Construct the where clause
+	params := make([]any, 0, 4)
+	where := make([]string, 0, 3)
+
+	params = append(params, sql.Named("orgID", orgID))
+	where = append(where, "organization_id=:orgID)")
+
+	if prevPage.EndIndex != "" {
+		var endIndex ulid.ULID
+		if endIndex, err = ulid.Parse(prevPage.EndIndex); err != nil {
+			return nil, nil, invalid(ErrInvalidCursor)
+		}
+
+		params = append(params, sql.Named("endIndex", endIndex))
+		where = append(where, "id > :endIndex")
+	}
```

Comment:
This part of the method is a bit hard for me to follow - can you add a few comments here explaining what's happening (at a high level) and why the query needs to be built like this?

Reply:
I have added a comment here as well as for pageSize and I added a note above the function to show the queries that would be executed.